packages/zoe/src/zoe.js

```diff
@@ -317,36 +361,20 @@ const makeZoe = (additionalEndowments = {}) => {
         return harden(redemptionResult);
```

Comment:
This should error if it's not `'Waived'`, rather than assuming it's covered all cases. And have an error if it's an unexpected value.

Reply:
Good point